skin/frontend/base/default/js/sd_recentlyviewed/recent.js

```diff
@@ -79,12 +79,17 @@ SomethingDigitalRecentlyViewed.prototype = {
 			return;
 		}
 
-		var html = rvTemplate.innerHTML.replace('{{items}}',this.getRenderedItems());
-		$(target).insert({top: html});
+		var insertParams = {}, //to enable use of variable keys (i.e. $position: content)
+			insertPosition = sdRecentlyViewed.insertPosition,
+			html = rvTemplate.innerHTML.replace('{{items}}',this.getRenderedItems());
+
+		insertParams[insertPosition] = html;
+
+		$(target).insert(insertParams);
```

Comment:
This requires it to be an ID; rather let's use `$$` and we can just grab the first item out. This way we can use arbitrary CSS selectors.